### PR TITLE
SB-1112: problem liquibase changeset fixed

### PR DESCRIPTION
**Pull request by sbespalov**
@steve-todorov you probably will need to drop our production DB again because I have fixed changeset which is already applied. This changes needed as an example to create valid changesets in the future.

### CODE REVIEW

File: strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
         <o:dropProperty className="AuthorizationConfig" propertyName="roles" />
 
+        <o:delete tableName="Roles"/>
```
Comment (sbespalov) [inline strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml]: @fuss86 you probably need to add same statement after dropping class properties. Seems that OrientDB is not assumed to drop data within dropping a property.

**Comment by steve-todorov** [comment]
Thanks for letting me know @sbespalov. Ping me again when this is merged so I can fix the instance :)

**Comment by sbespalov** [comment]
@carlspring can you please review?

File: strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml (hunk 1 above)
Comment (fuss86) [reply strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml]: @sbespalov thanks for this tip. Why have you used _o_ namespace prefix here ? _delete_ element comes from liquibase core as far as I see. Will it work prefixed with _o_ namespace ?

Comment (fuss86) [reply strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml]: And how can I use it to remove all values from the property it they were not wrapped by additional element (like `Repository#groupRepositories`) ?

Comment (fuss86) [reply strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml]: Maybe we should try to use standard liquibase _update_ data change element

**Pull request merged**